.gitignore

```diff
@@ -1,16 +1,7 @@
-.idea
```

Comment:
@joachimvh You might want to set a global `.gitignore` instead.

Reply:
I actually do, that one probably got there to be really safe 😄